### ISSUE TITLE
No support for MP4 files

### ISSUE THREAD

**Issue by TrueWinter** [NdT3Development/nodeupload#2]
Crashing when trying to view an MP4 video, probably no support for any video format, only images. May change this to be an image-only app due to complicated solutions that I have found for this issue.

**Comment by TrueWinter** [comment]
Error code in console on crash: 
![Error](https://ndt3.got-lost-in.space/5c66ab.png)

**Comment by TrueWinter** [comment]
May be an easy fix as this app should be run behind a proxy. Will require extra setup instructions. Probably only going to provide instructions for Nginx for this.

**Comment by TrueWinter** [comment]
After thinking about this issue, I have concluded that this will be an image only solution due to other issues that may come as a result of a fix _(these issues will **not** be discussed)_. There will also be too much time and effort involved in supporting users who come across these issues and in general will require too much work from my side. As always on my GitHub page, there are **no** warranties for **any** of the software in any of the repositories unless stated otherwise (usually in the `README.md` file). Also, unless otherwise stated, there is also **no** support given (repositories stating `Open an issue if...` or similar would have support **only** on GitHub unless otherwise stated in the repositories `README.md` file.

In short, I will not always fix all issues (this being one issue that will not be fixed) and I will not always give support.

Closing this issue and removing labels for the reasons above.

**Comment by TrueWinter** [comment]
I have thought of a way to fix this. Will be testing later. May break the whole app so do not use the testing version coming out later.

**Comment by TrueWinter** [comment]
Either use express.static or if that doesn't work, change to a whitelist based upload filter that sets headers according to file extension.

**Comment by TrueWinter** [comment]
I think I fixed it using express.static in a local version. Will continue testing. I'm not sure why I even tried to use sendFile for this...

**Comment by TrueWinter** [comment]
Fixed with support for most file types
